src/lib/operators/jit_operator/operators/jit_expression.cpp

```diff
@@ -1,13 +1,70 @@
-#include "jit_compute.hpp"
+#include "jit_expression.hpp"
+
+#include <string>
 
 #include "../jit_constant_mappings.hpp"
 #include "../jit_operations.hpp"
+#include "../jit_types.hpp"
 
 namespace opossum {
 
+// Returns the enum value (e.g., DataType::Int, DataType::String) of a data type defined in the DATA_TYPE_INFO sequence
+#define JIT_GET_ENUM_VALUE(index, s) APPEND_ENUM_NAMESPACE(_, _, BOOST_PP_TUPLE_ELEM(3, 1, BOOST_PP_SEQ_ELEM(index, s)))
```

Comment:
This looks familiar. Don't we have this in jit_operations.cpp, too?

Reply:
I removed `JIT_GET_ENUM_VALUE` and `JIT_GET_DATA_TYPE` macros from `jit_operations.hpp`, `jit_operations.cpp` and `jit_expressions.cpp` and added them to `jit_types.hpp` instead.